README.md

```diff
@@ -192,6 +192,36 @@ coverageReporter: {
 }
 ```
 
+#### watermarks
+**Type:** Object
+
+**Description:** This will be used to set the coverage threshold colors. The first number is the threshold between Red and Yellow. The second number is the threshold between Yellow and Green.
+
+```javascript
+coverageReporter: {
```

Comment:
No, they don't, this was a merge conflict that I resolved when I merged in master.